styles.css

```
@@ -1,15 +1,65 @@
-* .header {
-  background: #140523;
-  color: #ffffff;
-  height: 100%;
-  box-shadow: rgba(0, 0, 0, 0.24) 0 3px 8px;
-  padding: 10px;
+body {
+  background-color: rgb(233, 243, 240);
+}
+
+.header {
+  left: 0;
+  height: 70px;
+  width: 100%;
+  border: 2px solid black;
+  top: 0;
   position: fixed;
+  display: flex;
+  justify-content: center;
+  z-index: 10;
+  align-items: center;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.QuestionBox {
+  list-style: none;
+  margin-top: 180px;
+  border: 1px solid black;
+  padding: 50px;
+
+  text-align: center;
+}
+
+.options {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 8px;
+  color: black;
+  list-style: none;
+  border-radius: 2px;
 }
 
 .footer {
-  background: #140523;
-  color: #ffffff;
-  height: 100%;
+  color: black;
+  width: 100%;
+  height: 50px;
+  bottom: 0;
   position: fixed;
+  border-top: 1px solid black;
+
+  align-items: center;
+  flex-direction: row;
+}
+
+.footerul {
```

this appears to be missing a space:

```suggestion
.footer ul {
```